iwyu_output.h

```diff
@@ -213,7 +215,7 @@ class IwyuFileInfo {
                   const string& quoted_includee, int linenumber);
   // definitely_keep_fwd_decl tells us that we should never suggest
   // the fwd-decl be removed, even if we don't see any uses of it.
-  void AddForwardDeclare(const clang::NamedDecl* forward_declare_decl,
+  void AddForwardDeclare(const clang::NamedDecl* fwd_decl,
```

Comment:
Here I think it is better to sync implementation with declaration, not vice versa. Mostly because I think that this argument is not forward declaration per se but a NamedDecl that can be forward declared.